baiji/cli.py

```diff
@@ -19,7 +19,7 @@ class ListCommand(BaijiCommand):
     DESCRIPTION = "list files on s3"
     uri = cli.Flag(["-B", "--uri"], help='This option does nothing. It used to return URIs instead of paths, but this is now the default.')
     detail = cli.Flag(['-l', '--detail'], help='print details, like `ls -l`')
-    shallow = cli.Flag("--shallow", help='process key names hierarchically and return only immediate "children" (like ls, instead of like find)')
+    shallow = cli.Flag(['-s', "--shallow"], help='process key names hierarchically and return only immediate "children" (like ls, instead of like find)')
     list_versions = cli.Flag(['--list-versions'], help='print all versions')
```

Comment:
hahaha, after all these years, you finally got sick of typing out '--shallow'?? :)